types/dom-webcodecs/webcodecs.generated.d.ts

```diff
@@ -137,7 +137,7 @@ interface VideoEncoderConfig {
     alpha?: AlphaOption | undefined;
     avc?: AvcEncoderConfig | undefined;
     bitrate?: number | undefined;
-    bitrateMode?: BitrateMode | undefined;
+    bitrateMode?: VideoEncoderBitrateMode | undefined;
```

Comment:
I was going to say that you can just revert the BitrateMode change from https://github.com/DefinitelyTyped/DefinitelyTyped/commit/7e99574c3ba0a7e13c1c8718971da60f408bbe97 but it looks like Nathan bumped it to TS 5.1+ anyhow, so, I guess this is fine.

Reply:
Not sure what criteria we use to distinguish when we should bump the minimum, versus copying and using typesVersions...

Reply:
I dont know either.